evm/its.js

```diff
@@ -419,15 +430,32 @@ async function processCommand(_axelar, chain, chains, action, options) {
                 return;
             }
 
-            const data = [];
+            if (itsVersion === '2.1.1') {
+                const trustedAddresses = [];
+                for (const trustedChain of trustedChains) {
+                    const chainConfig = chains[trustedChain];
+                    if (chainConfig?.contracts?.InterchainTokenService?.address) {
+                        trustedAddresses.push(chainConfig.contracts.InterchainTokenService.address);
+                    } else {
+                        throw new Error(`No InterchainTokenService address found for chain ${trustedChain}`);
+                    }
+                }
+                
+                for (const [trustedChain, trustedAddress] of trustedChains.map((chain, index) => [chain, trustedAddresses[index]])) {
+                    const tx = await interchainTokenService.setTrustedAddress(trustedChain, trustedAddress, gasOptions);
+                    await handleTx(tx, chain, interchainTokenService, action, 'TrustedAddressSet');
+                }
+            } else {
+                const data = [];
 
-            for (const trustedChain of trustedChains) {
-                const tx = await interchainTokenService.populateTransaction.setTrustedChain(trustedChain, gasOptions);
-                data.push(tx.data);
-            }
+                for (const trustedChain of trustedChains) {
+                    const tx = await interchainTokenService.populateTransaction.setTrustedChain(trustedChain, gasOptions);
```

Comment:
only this line needs to be different depending on the version, not the full loop